kubernetes/spec/support/e2e/coverage_reporter.rb

```
@@ -0,0 +1,90 @@
+          file.write(JSON.pretty_generate(data))
+          file.flush
+          file.fsync
+        end
+
+        File.rename(temp_path, output_path)
+        output_path
+      end
+
+      private
+
+      def default_output_path(output_dir)
```

RuboCop (1.85.1)

[convention] Use 2 (not 0) spaces for indented_internal_methods indentation.

(Layout/IndentationWidth)